pontoon/base/utils.py

```diff
@@ -21,10 +21,13 @@
 from django.db.models import Prefetch, Q
 from django.db.models.query import QuerySet
 from django.http import HttpResponseBadRequest
-from django.shortcuts import get_object_or_404
 from django.utils import timezone
 from django.utils.text import slugify
 from django.utils.translation import trans_real
+from django.shortcuts import redirect, get_object_or_404
+from django.apps import apps
+from django.urls import reverse
+from django.http import Http404
```

Comment:
nitpick: Would be nice to keep the imports sorted and un-duplicated, but I don't think we've been very strict about that.